src/mtcp/mtcp_restart.c

```diff
@@ -710,8 +714,6 @@ main(int argc, char *argv[], char **environ)
     // If we want to test this, we can add code to do a trial mremap with a page
     //   before vvar and after vdso, and verify that we get an EFAULT.
     // In May, 2020, on Cori and elsewhere, vvar is 3 pages and vdso is 2 pages.
-    char *vdsoStart = (char *)mygetauxval(environ, AT_SYSINFO_EHDR);
```

Comment:
What is this for and why removing this line?

Reply:
this is a currently unused variable that was previously used when determining where to temporarily map the `vdso`/`vvar` segments to. there used to be a function `mysetauxval` that set `AT_SYSINFO_EHDR` so this line could read for it, but since that function is no longer in use we can remove this one as well.